### PR TITLE
[tests-only] Don't error if lock is not found

### PR DESCRIPTION
**Pull request by kobergj**
Is supposed to fix failing pipelines like this: https://drone.owncloud.com/owncloud/ocis/9533/25/4

### CODE REVIEW

File: pkg/storage/utils/decomposedfs/xattrs/xattrs.go [hunk 1]
```diff
@@ -130,7 +130,7 @@ func CopyMetadata(src, target string, filter func(attributeName string) bool) (e
 		rerr := filelocks.ReleaseLock(writeLock)
 
 		// if err is non nil we do not overwrite that
-		if err == nil {
+		if err == nil && !isNotFoundError(rerr) {
```
Comment (C0rby) [inline pkg/storage/utils/decomposedfs/xattrs/xattrs.go:133]:
```suggestion
		if err == nil && !errors.Is(rerr, fs.ErrNotExist) {
```

**Review by C0rby — CHANGES_REQUESTED** [review]

File: pkg/storage/utils/decomposedfs/xattrs/xattrs.go (hunk 1 above)
Comment (kobergj) [reply pkg/storage/utils/decomposedfs/xattrs/xattrs.go]: Much better :+1:

**Comment by dragotin** [comment]
Oh no, please don't. This is hiding a real problem, which is: There are two processes trying to create a file. There was a bug in the locking code that failed to detect that and wait a moment until the first one has released.

That was fixed in 59a37c326a5b2ccdcbfb95a5174665017b7ea0bb . Please double check if you have that commit, and if you have, we need to investigate...

**Review by dragotin — CHANGES_REQUESTED** [review]
Please dont merge. See comment.

**Comment by C0rby** [comment]
Right, I think we should do an error check before `ReleaseLock`. If the lock couldn't be set then we should try to release it. But still return the original error.

**Pull request closed without merging**